### ISSUE TITLE
Keyboard support & proper labeling of things

### ISSUE THREAD

**Issue by Freeplayg** [transfem-org/Sharkey#80]
## Summary

Am not sure if this should count as a bug or feature, but having keyboard support, as well as properly labeling buttons for screen readers should be a needed thing. 

I know one way most of the labeling was done in Firefish was setting the aria-label attribute in the script that adds the tooltips, so that should make that bit easier. Some cherry-picking could also probably be done for especially the form components i think? I hope I didn't make those commits too messy, I'll try to find them later

Buttons should also have an outline shown when they're focused

There's a bunch more stuff that would need to be done, but that's the most basic stuff to make it at least _sort of_ usable. Am still learning this stuff also

**Comment by Kio-td** [comment]
Related: #23

**Comment by Mar0xy** [comment]
"but having keyboard support" don't really what is meant with this as you can scroll using the keyboard as well as switch light/dark mode, open new post dialog

**Comment by Freeplayg** [comment]
keyboard support as in being able to tab through every button, basic accessibility
